nltk/corpus/__init__.py

```diff
@@ -482,7 +482,7 @@ def demo():
     # demo()
     pass
 
-# ** this is for nose **
+# ** this is for unit testing **
```

Comment:
One potential next step would be to turn this into a pytest fixture and move it into the testing code as opposed to having it in "production" code.

This applies to the `setup_module` function as well.